src/components/Announcement/Announcement.jsx

```diff
@@ -16,21 +17,25 @@ function Announcement() {
   };
 
   return (
-    <div className="announcement-banner">
+    <div className="announcement-banner ohm-card">
       {newNetworkVisible && (
         <>
-          <Paper className="new-network" style={isMobileScreen ? { borderRadius: "0" } : { width: "100%" }}>
```

Comment:
if we use ohm-card for the outer wrapper we will need to preserve the 0 border radius on mobile so the edges go to the edge of the screen